src/lib/bootloader/systeminfo.rb

```diff
@@ -62,6 +91,9 @@ def trusted_boot_active?
 
       # Check if the system is expected to have nvram - ie. update_nvram_active? makes a difference
       def nvram_available?(bootloader_name = nil)
+        # not for grub2-bls
+        return false if bootloader_name == "grub2-bls"
```

Comment:
@aplanas Just to be sure, grub2-bls cannot skip writing to nvram? I remember that especially some aarch64 devices has problems to write to it and needs this "workaround" which skips write to it, otherwise a crash happen. I can find some bugzilla references, if you are interested.